geopandas/io/arrow.py

```diff
@@ -280,12 +285,25 @@ def _to_parquet(df, path, index=None, compression="snappy", version=None, **kwar
         "pyarrow.parquet", extra="pyarrow is required for Parquet support."
     )
 
+    if kwargs and "version" in kwargs and kwargs["version"] is not None:
+        warnings.warn(
```

Comment:
We should only raise the warning here if additionally `version in {"0.1.0", "0.4.0"}` ?